src/main/java/frc/team2412/robot/commands/hood/HoodSetAngleCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.team2412.robot.commands.hood;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystems.HoodSubsystem;
+import frc.team2412.robot.subsystems.constants.ShooterConstants.ShooterDistanceDataPoint;
+
+public class HoodSetAngleCommand extends CommandBase {
+
+	Supplier<ShooterDistanceDataPoint> m_distanceSupplier;
+	HoodSubsystem m_HoodSubsystem;
+
+	public HoodSetAngleCommand(HoodSubsystem m_HoodSubsystem, Supplier<ShooterDistanceDataPoint> m_distanceSupplier) {

```

Comment:
Won't the hood angle depend on lift height (especially when close)?  Am I missing where this is taken into consideration?